paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionUi.kt

```diff
@@ -62,136 +66,192 @@ internal fun PaymentOptionUi(
+                            // one one one one vs one thousand one hundred eleven
+                            this.contentDescription =
+                                description.replace("\\d".toRegex(), "$0 ")
+                        }
                 )
             }
-        }
-        if (isEditing && onRemoveListener != null) {
-            val openDialog = remember { mutableStateOf(false) }
+        },
+        modifier = modifier
+            .padding(top = 12.dp)
+            .requiredWidth(viewWidth)
+            .alpha(alpha = if (isEnabled) 1.0F else 0.6F)
+    )
 
-            SimpleDialogElementUI(
-                openDialog = openDialog,
-                titleText = removePmDialogTitle,
-                messageText = description,
-                confirmText = stringResource(R.string.remove),
-                dismissText = stringResource(R.string.cancel),
-                onConfirmListener = onRemoveListener
-            )
+    if (isEditing && onRemoveListener != null) {
+        SimpleDialogElementUI(
+            openDialog = openRemoveDialog,
+            titleText = removePmDialogTitle,
+            messageText = description,
+            confirmText = stringResource(R.string.remove),
+            dismissText = stringResource(R.string.cancel),
+            onConfirmListener = onRemoveListener
+        )
+    }
+}
 
-            // tint the delete symbol so it contrasts well with the error color around it.
-            val iconColor = MaterialTheme.colors.error
-            val deleteIconColor = if (iconColor.shouldUseDarkDynamicColor()) {
-                Color.Black
-            } else {
-                Color.White
-            }
+@Composable
+private fun PaymentOptionCard(
+    isSelected: Boolean,
+    isEnabled: Boolean,
+    iconRes: Int,
+    iconTint: Color?,
+    labelText: String,
+    onItemSelectedListener: (() -> Unit),
+    modifier: Modifier = Modifier,
+) {
+    SectionCard(
+        isSelected = isSelected,
+        modifier = modifier
+            .height(64.dp)
+            .padding(horizontal = 6.dp)
+            .fillMaxWidth()
+    ) {
+        Column(
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .fillMaxSize()
+                .testTag("${PAYMENT_OPTION_CARD_TEST_TAG}_$labelText")
+                .selectable(
+                    selected = isSelected,
+                    enabled = isEnabled,
+                    onClick = onItemSelectedListener,
+                ),
+        ) {
             Image(
-                painter = painterResource(R.drawable.stripe_ic_delete_symbol),
-                contentDescription = onRemoveAccessibilityDescription,
-                colorFilter = ColorFilter.tint(deleteIconColor),
+                painter = painterResource(iconRes),
+                contentDescription = null,
```

Comment:
Where did this move to? I see it in code, but would be helpful for my context if you could explain conceptually.

Reply:
Sure. The main card content was moved into `PaymentOptionCard`, which is placed in a `Column` together with `LpmSelectorText`. The accessory views (remove badge, selection badge) were moved into their own composable functions and are passed into the `BadgedBox` function as a parameter.